test/spec/Garn/CodeGenSpec.hs

```diff
@@ -5,249 +5,242 @@ module Garn.CodeGenSpec (spec) where
 import Data.Map (Map, singleton)
 import Data.String.Interpolate (i)
 import Data.String.Interpolate.Util (unindent)
-import GHC.Conc (getNumProcessors, setNumCapabilities)
-import Garn.CodeGen (PkgInfo (Collection, Derivation, description, path, subPkgs), scanPackages, writePkgFiles)
+import Garn.CodeGen (PkgInfo (Collection, Derivation, description, path), scanPackages, writePkgFiles)
 import Garn.Common (currentSystem)
 import System.FilePath ((</>))
 import System.IO.Temp (withSystemTempDirectory)
 import Test.Hspec
 import Test.Mockery.Environment (withModifiedEnvironment)
 
-inParallel :: Spec -> Spec
-inParallel spec = do
-  runIO $ do
-    n <- getNumProcessors
-    setNumCapabilities (max 1 (n - 1))
-  parallel spec
-
 spec :: Spec
-spec = inParallel $ do
+spec = parallel $ do
```

Comment:
As discussed, I reverted this parallel change because it wasn't working on GitHub actions.